django/test/utils.py

```diff
@@ -463,22 +463,26 @@ def inner(*args, **kwargs):
 
 
 @contextmanager
-def patch_logger(logger_name, log_level):
+def patch_logger(logger_name, *log_levels):
```

Comment:
Given that I don't have multiple log levels anymore, I leave this file out of the merge.